src/clients/Elsa.Api.Client/Shared/Models/TimestampFilter.cs

```diff
@@ -0,0 +1,26 @@
+using Elsa.Api.Client.Shared.Enums;
+using JetBrains.Annotations;
+
+namespace Elsa.Api.Client.Shared.Models;
+
+/// <summary>
+/// Represents a timestamp filter used for filtering data based on a specified timestamp column and operator.
+/// </summary>
+[UsedImplicitly]
+public class TimestampFilter
+{
+    /// <summary>
+    /// Gets or sets the column to filter by.
+    /// </summary>
+    public string Column { get; set; } = default!;
```

Comment:
Should not be an enum like 
{
CreatedAt,
UpdatedAt,
FinishedAt,
}

or add a validator ?

Reply:
I considered adding an enum instead of a string, but figured that we might keep the `TimestampFilter` more general purpose, instead of only applying to the `WorkflowInstance` entity.

But I agree with your second suggestion and add a validator at the WorkflowInstance List API endpoint and return a 400 Bad Request in case the client sends an invalid column name (instead of a 500 internal server error that would happen right now) 👍🏻